Storage/Rust/RustPlaces.swift

```diff
@@ -0,0 +1,287 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+@_exported import MozillaAppServices
+
+private let log = Logger.syncLogger
+
+public class RustPlaces {
```

Comment:
Was this accidental? This file shouldn't be added here otherwise we're gonna collide when we merge the `rust-bookmarks` branch.